doc/users/next_whats_new/2017-12-05_share_unshare_axes_after_creation.rst

```diff
@@ -0,0 +1,5 @@
+Share and unshare `axes` after creation
+---------------------------------------
+
+`Axes` have `unshare_x_axes`, `unshare_y_axes`, `unshare_z_axes` and `unshare_axes` methods to unshare axes.
+Similiar there are `share_x_axes`, `share_y_axes`, `share_z_axes` and `share_axes` methods to share axes.
```

Comment:
```
`~.Axes` have `~.Axes.unshare_x_axes`, `~.Axes.unshare_y_axes`, `~.Axes.unshare_z_axes` and `~.Axes.unshare_axes` methods to unshare axes.  Similarly, there are `~.Axes.share_x_axes`, `~.Axes.share_y_axes`, `~.Axes.share_z_axes` and `~.Axes.share_axes` methods to share axes.
```

I would also incldue a short description of what happens when you unshare an axis.  I assume the viewlimits etc stay the same, but are no longer coupled to further changes.